### PR TITLE
[DOCFIX]Add java doc for tests/src/test/java/alluxio/collections/IndexedSetCo…

### PR DESCRIPTION
**Pull request by Popeye13**
…ncurrencyTest.java

### CODE REVIEW

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java [hunk 1]
```diff
@@ -99,6 +102,9 @@ public long runSingleTask() {
     }
   }
 
+  /**
+   * Adds a concurrent task with check.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: What does "with check" mean?

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java [hunk 2]
```diff
@@ -122,6 +128,9 @@ public long runSingleTask() {
     }
   }
 
+  /**
+   * Removes a concurrent task.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: I don't think this is correct

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java [hunk 3]
```diff
@@ -88,6 +88,9 @@ public Void call() throws BrokenBarrierException, InterruptedException {
     }
   }
 
+  /**
+   * Adds a concurrent task.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: This is a class, not a method

**Comment by Popeye13** [comment]
@aaudiber Thanks for your review, updated.

**Comment by aaudiber** [comment]
@Popeye13 The build failed because one of the lines is over 100 characters

**Pull request merged**